applications/DEMApplication/tests/restart_files/two_ballsProjectParametersDEM.json

```diff
@@ -0,0 +1,82 @@
+    "BoundingBoxMinX"                : -10,
+    "BoundingBoxMinY"                : -10,
+    "BoundingBoxMinZ"                : -10,
+    "dem_inlet_option"               : false,
+    "GravityX"                       : 0.0,
+    "GravityY"                       : 0.0,
+    "GravityZ"                       : -9.81,
+    "RotationOption"                 : true,
+    "CleanIndentationsOption"        : false,
+    "solver_settings"                : {
+        "RemoveBallsInitiallyTouchingWalls" : false,
+        "strategy"                          : "sphere_strategy",
+        "model_import_settings"           : {
+            "input_type"     : "mdpa",
+            "serializer_trace"             : "trace_all",
+            "input_filenames" : ["SpheresPart"],
+            "restart_load_file_label" : "0.03001"
+        }
+    },
+    "VirtualMassCoefficient"         : 1.0,
+    "RollingFrictionOption"          : false,
+    "GlobalDamping"                  : 0.0,
+    "ContactMeshOption"              : false,
+    "OutputFileType"                 : "Binary",
+    "Multifile"                      : "multiple_files",
+    "ElementType"                    : "SphericPartDEMElement3D",
+    "TranslationalIntegrationScheme" : "Symplectic_Euler",
+    "RotationalIntegrationScheme"    : "Direct_Integration",
+    "MaxTimeStep"                    : 1e-5,
+    "FinalTime"                      : 0.03005,
+    "GraphExportFreq"                : 0.001,
+    "VelTrapGraphExportFreq"         : 0.001,
+    "OutputTimeStep"                 : 1e-5,
+    "PostBoundingBox"                : false,
+    "PostLocalContactForce"          : false,
+    "PostDisplacement"               : true,
+    "PostRadius"                     : true,
+    "PostVelocity"                   : true,
+    "PostAngularVelocity"            : true,
+    "PostElasticForces"              : true,
+    "PostContactForces"              : true,
+    "PostRigidElementForces"         : false,
+    "PostStressStrainOption"         : false,
+    "PostTangentialElasticForces"    : false,
+    "PostTotalForces"                : true,
+    "PostPressure"                   : false,
+    "PostShearStress"                : false,
+    "PostNonDimensionalVolumeWear"   : false,
+    "PostParticleMoment"             : true,
+    "PostEulerAngles"                : false,
+    "PostRollingResistanceMoment"    : false,
+    "do_print_results_option"        : false,
+    "problem_name"                   : "two_balls",
+
+    "output_processes" : {
+        "restart_processes" : [
+            {
+            "python_module"   : "DEM_save_restart_process",
+            "kratos_module"   : "KratosMultiphysics",
+            "process_name"    : "DEMSaveRestartProcess",
+            "Parameters"            : {
+                "model_part_names" : ["SpheresPart"],
+                "echo_level"                   : 0,
+                "serializer_trace"             : "trace_all",
```

Comment:
I would use `no_trace` for the tests, is faster
There are separate tests for this

Reply:
yes, it was a leftover debugging setting